src/main/java/frc/robot/AutoGroup.java

```diff
@@ -0,0 +1,31 @@
+/* ==================================================
+ * Authors: Lucas Jacobs
+ * --------------------------------------------------
+ * Description:
+ * This class caontinds the commands that are executed 
+ * in autonomous mode at the start of the match
+ * ================================================== */
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.auto.*;
+
+public class AutoGroup extends SequentialCommandGroup {
+    public AutoGroup(Robot robot) {
```

Comment:
AutoGroup is kinda generic, what if you have multiple auto routines depending on where you are on the field?  Maybe something like `ReverseAndShoot`

Reply:
Good point, I'll change that in another PR.